### ISSUE TITLE
Adding virtualenv_path_extensions.pth through add2virtualenv should auto show up in search paths

### ISSUE THREAD

**Issue by sonictk** [Microsoft/PTVS#863]
Hi PTVS:

Right now, I have a virtual environment that I'm adding search paths to via using `add2virtualenv`, which essentially creates a file called `virtualenv_path_extensions.pth` in the \lib\site-packages folder of the virtual environment with the paths to the folders in questions that I want to add to the search path. This works out great for developers, because then they can use their IDE of choice and still use the same virtual environment setup.

However, it seems like PTVS, when adding existing virtual environments, does not seem to do this automatically (i.e. propogating the Search Paths by looking at path configuration files as well). 

I feel like this would be a great quality of life feature to add to PTVS if it isn't too difficult to implement, as PyCharm and Wing both support this when adding virtual environments, which makes controlling a development environment across multiple workstations significantly easier; what do other people think about this?

**Comment by zooba** [comment]
It's something that we added for regular environments in 2.2, but not for virtual environments. The reason we didn't add it there is because it would have broken the layering we've already implemented in our database, basically forcing every virtual environment to do a full reanalysis of the library instead of the quick one it can do now.

Long term, we want to replace this database format completely with one that will handle this sort of thing properly, and as I said we've already made moves towards using Python to find library directories (and hence .pth files will be handled consistently) rather than guessing ourselves. This may come up in PTVS 3.0, depending on the usual scheduling concerns, but it is certainly something that I personally am keen to do (which makes it much more likely to happen :) ).
